functions.php

```diff
@@ -2,6 +2,58 @@
 
 function my_theme_enqueue_styles() {
     wp_enqueue_style( 'my-theme-style', get_stylesheet_uri() );
- wp_enqueue_style('my-theme-extra-style', get_theme_file_uri('extra.css') );
+    wp_enqueue_style('normalize', get_theme_file_uri('/css/normalize.css') );
 }
 add_action( 'wp_enqueue_scripts', 'my_theme_enqueue_styles' );
+
+function portfolio_customizer_settings($wp_customize) {
+    //section
+    $wp_customize->add_section('portfolio_settings', array(
+        'title' => __('Portfolio Settings', 'theme-textdomain')
```

Comment:
Aqui va el que creaste para tu tema en style.css
```suggestion
        'title' => __('Portfolio Settings', 'aprendiendoezequiel')
```